### DISCUSSION TITLE
What does Serverless Functions per Month mean ?

### DISCUSSION THREAD

**Discussion by DiamondYuan** [vercel/vercel#5281] [Help] [answered]
I have a project with 10 Serverless Functions , they use the same middleware.
If i change the middleware,it will detract 10 from the available limit ?

**Comment by mcsdevv** [comment, marked as answer]
Hello @DiamondYuan, it means any files that get turned into Serverless Functions.

So if your middleware changes, not a problem, the content of your Serverless Functions is not impacted by this.

Only when the content of this Serverless Function file itself changes, would the limit be subtracted from.
